modules/installation-osp-creating-image.adoc

```diff
@@ -19,7 +19,12 @@ The {product-title} installation program requires that a {op-system-first} image
 
 . Under *Version*, select version 4.2.0 for RHEL 8.
 
-. Download the _Red Hat Enterprise Linux CoreOS - OpenStack Image (QCOW)_.
+. Download the _Red Hat Enterprise Linux CoreOS - OpenStack Image (QCOW)_ and decompress the image..
++
+[NOTE]
+====
+The OpenStack image must be decompressed before being used by the cluster.
```

Comment:
"You must decompress the OpenStack image before the cluster can use it."

Otherwise, :+1: